src/query/plan/operator.hpp

```diff
@@ -1392,6 +1392,8 @@ class Delete : public memgraph::query::plan::LogicalOperator {
   /// Whether the vertex should be detached before deletion. If not detached,
   ///            and has connections, an error is raised when deleting edges.
   bool detach_;
+  // when buffer size is reached, delete will be triggered
+  Expression *buffer_size_;
```

Comment:
I would call it  periodic_commit_limit, as it is from periodic commit, so it makes sense to be tied together here

Reply:
I was more thinking like, maybe something else in the future can trigger the buffer size to be different?

Reply:
It sounds to me like premature optimization, but I will leave it up to you